client/app/hosting/framework/runtime/edit/hosting-framework-runtime-edit.html

```diff
@@ -0,0 +1,69 @@
+<div data-ng-controller="controllers.Hosting.Framework.Runtime.edit as ctrl">
+    <div data-wizard
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="edit"
+         data-wizard-title="i18n.hosting_tab_FRAMEWORK_runtime_edit_title">
+
+        <div data-wizard-step
+             data-wizard-step-valid="isValid()">
```

Comment:
ctrl.isValid() and like @FredericEspiau said, there would be a more Angular way to validate a form.